.travis.yml

```diff
@@ -3,9 +3,11 @@ language: generic
 env:
   global:
     - ENCRYPTION_LABEL="e149beb9c312"
-    - DEPLOY_USER="annevankesteren"
 script:
   - curl --remote-name --fail https://resources.whatwg.org/build/deploy.sh && bash ./deploy.sh
+branches:
+  only:
+    - master
```

Comment:
Does this not mean that we no longer validate PRs?

Reply:
No, PRs are separate. It means that PRs from this repo (instead of from forks) only run Travis once (for the PR), not twice (once for the PR and once for the branch).